inst/testthat/helper_autotest.R

```diff
@@ -1,17 +1,37 @@
-# Learner autotest suite
-#
-# `run_experiment(task, learner)` runs a single experiment.
-# Returns a list with success flag "status" (`logical(1)`),
-# "experiment" (partially constructed experiment), and "error"
-# (informative error message).
-#
-# `run_autotest(learner)` generates multiple tasks, depending on the properties of the learner.
-# and tests the learner on each task, with each predict type.
-# To debug, simply run `result = run_autotest(learner)` and proceed with investigating
-# the task, learner and prediction of the returned `result`.
+#' @title Learner Autotest Suite
+#'
+#' @description
+#' The autotest suite is a collection of functions to test learners in a standardized way.
+#' Extension packages need to specialize the S3 methods in the file.
 #
-# NB: Extension packages need to specialize the S3 methods in the file.
+#' @details
+#' `run_autotest(learner)` generates multiple tasks, depending on the properties of the learner and tests the learner on each task, with each predict type.
+#' Calls `generate_tasks()` to generate tasks and `run_experiment()` to run the experiments.
+#' To debug, simply run `result = run_autotest(learner)` and proceed with investigating he task, learner and prediction of the returned `result`.
+#'
+#' `run_experiment(task, learner)` runs a single experiment.
+#' Calls `train()` and `predict()` on the learner and checks the prediction with `score()`.
+#' The prediction is checked with `sanity_check()`.
+#'
+#' `generate_tasks(learner)` generates multiple tasks for a given learner.
+#' Calls `generate_data()` and `generate_generic_tasks()` to generate tasks with different feature types.
+#'
+#' @noRd
+NULL
 
+#' @title Generate Tasks for a Learner
```

Comment:
would be nice to add a few lines here a) about the result. i guess it is a list named by properties?
b) can we at least SUMMARIZE what tasks are created? BRIEFLY?

Reply:
Done